Flutter User Facing API.dart

```diff
@@ -0,0 +1,71 @@
+class Manager {
+  Future<Null> createClient(String restoreStateIdentifier) async
+  Future<Null> destroyClient() async
+
+  Future<Null> enableRadio()
+  Future<Null> disableRadio()
+  
+  Future<Null> cancelTransaction(String transactionId) async
+
+  Future<Null> setLogLevel(LogLevel logLevel)
+  Future<LogLevel> logLevel()
+
+  Future<BluetoothState> state()
+  Stream<BluetoothState> onStateChange()
+
+  Stream<ScanResult> startDeviceScan(int scanMode, int callbackType, List<String> uuids) async*
+  Future<Null> stopDeviceScan() async
+
+  Future<List<Device>> devices(String[] ids)
+  Future<List<Device>> connectedDevicesWithServices(String[] serviceUUIDs)
+}
+
+class Device {
+  Future<BleDevice> connect({ bool isAutoConnect, int requestMtu }) async
+  Future<bool> isConnected() async
+  Stream<BleDevice> onConnectionChanged()
+  Future<BleDevice> cancelConnection() async
+
+  Future<bool> requestMTU(int mtu, String transactionId) async
```

Comment:
Was this intended? It is not available on iOS directly

Reply:
So it always fails on iOS and does some logic on Android. Contract's the same for both platforms.